Marlin/src/gcode/host/M115.cpp

```diff
@@ -295,6 +307,21 @@ void GcodeSuite::M115() {
       );
     #endif
 
+    // Temperatures
+    #if ENABLED(M115_TEMPERATURE_REPORT)
+      SERIAL_ECHOLNPGM(
+        "temperatures:{"
+          "t0:{min:", HEATER_0_MINTEMP, ",max:", HEATER_0_MAXTEMP, "}"
+          #if TEMP_SENSOR_1
+          ",t1:{min:", HEATER_1_MINTEMP, ",max:", HEATER_1_MAXTEMP, "}"
+          #endif
+          #if TEMP_SENSOR_BED
+          ",bed:{min:", BED_MINTEMP, ",max:", BED_MAXTEMP, "}"
+          #endif
+        "}"
+      );
+    #endif
+
```

Comment:
Marlin supports up to 8 hotends as well as a chamber, so they should be accounted for as well.

Reply:
What about the other devices with temperatures? You still have `TEMP_SENSOR_PROBE`, `TEMP_SENSOR_CHAMBER`, `TEMP_SENSOR_COOLER`, `TEMP_SENSOR_BOARD`, `TEMP_SENSOR_SOC`, and `TEMP_SENSOR_REDUNDANT`.

Reply:
Ok. I will add these.

Reply:
Only CHAMBER has MIN and MAX temps in addition to hotends and bed.

Reply:
> Only CHAMBER has MIN and MAX temps in addition to hotends and bed.

They're just not in the same place. Example: https://github.com/MarlinFirmware/Marlin/blob/9a1c993719b30dabfb2ab62101ba249dffa459c7/Marlin/Configuration_adv.h#L287

Reply:
Cooler and Board have both. SOC has only max temp. Redundant / Probe have neither.